wwwroot/sign-in.txt

```diff
@@ -1,6 +1,4 @@
-SIGN IN SHEET FOR GITFLOW
-Done When: Your LAST name and github username are listed below the last name on the list below 
-        
```

Comment:
Shelby - please leave this title and "Done When: " info in the file. 

Also correct your entry to meet the Done When requirements, e.g., Last name only